lnd_test.go

```diff
@@ -236,6 +236,30 @@ func assertNumChannelsPending(ctxt context.Context, t *harnessTest,
 	}
 }
 
+//assertNumConnections asserts number current connections between two peers
+func assertNumConnections(ctxt context.Context, t *harnessTest,
+	alice, bob *lightningNode, expected int) {
+	aliceNumPeers, err := alice.ListPeers(ctxt, &lnrpc.ListPeersRequest{})
+	if err != nil {
+		t.Fatalf("error fetching alice's node (%v) list peers %v",
```

Comment:
`error fetching` -> `unable to fetch`, the same logic should be applied to other error messages.